### PR TITLE
Production Push

### PR DESCRIPTION
**Pull request by idmken**

### CODE REVIEW

**Review by idmken — COMMENTED** [review]
Looks good.

File: _buy/fips201apl-pacs.md [hunk 1]
```diff
@@ -46,66 +46,19 @@ Visit the [Buy Page](../) to view FICAM products, services and purchasing guidan
...
-                {% if guide.reader13 %}
-                  <li><a href="{{ guide.reader8url | prepend: site.baseurl }}" target="_blank">{{ guide.reader13 }}</a></li>
-                {% endif %}
-                {% if guide.reader14 %}
-                  <li><a href="{{ guide.reader9url | prepend: site.baseurl }}" target="_blank">{{ guide.reader14 }}</a></li>
-                {% endif %}
-                </ol>
-              </ul>
-            </td>
+             <td headers="pacs-table-heading-{{ category | slugify }} pacs-table-heading-valapl">{{ guide.valapl }}</td>
```
Comment (idmken) [inline _buy/fips201apl-pacs.md:61]: @maxwellfunk what happens with the card reader pdfs? Do they stay on the site or remove them? They should be linked somewhere.

Comment (maxwellfunk) [reply _buy/fips201apl-pacs.md:61]: Currently developing a new system for linking to the card reader APLs... branch link below:
https://github.com/GSA/idmanagement.gov/tree/Reader_list_addition

**Review by JBPayne007 — APPROVED** [review]
Thanks

**Pull request merged**